pkg/apis/core/validation/validation.go

```diff
@@ -4351,10 +4352,24 @@ func ValidateConfigMap(cfg *core.ConfigMap) field.ErrorList {
 		for _, msg := range validation.IsConfigMapKey(key) {
 			allErrs = append(allErrs, field.Invalid(field.NewPath("data").Key(key), key, msg))
 		}
+		// check if the value stored consists entirely of valid UTF-8-encoded runes
+		if !utf8.ValidString(cfg.Data[key]) {
+			allErrs = append(allErrs, field.Invalid(field.NewPath("data").Key(key), key, field.ErrorTypeInvalid.String()))
```

Comment:
insteadof `field.ErrorTypeInvalid.String()`, indicate the data is not valid UTF-8 data

Reply:
Done